web/src/components/demo_pages/Intro/intro.css

```diff
@@ -0,0 +1,91 @@
+@import url('../../vars.css');
+
+.container {
+    height: 100vh;
+    width: 100vw;
+    padding-top: 2.5vh;
+    background: linear-gradient(180deg, #f3f2f1 0%, #ffffff 101.23%);
+}
+
+.text-box {
+    display: flex;
+    flex-direction: column;
+    padding: 0px;
+    height: 173px;
+    left: calc(50% - 700px / 2 + 1px);
+    bottom: 68px;
+}
+
+.text > h1 {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 36px;
```

Comment:
Not a big deal, but this is an example of what I mean about not worrying about being pixel perfect. Figma might specify this as 36px, but our typical largest font is 34px according to `vars`, and I would prefer to err on the side of consistency within the site rather than creating edge cases for every new design.